pkg/schemadsl/dslshape/dslshape.go

```diff
@@ -11,6 +11,7 @@ const (
 	NodeTypeError   NodeType = iota // error occurred; value is text of error
 	NodeTypeFile                    // The file root node
 	NodeTypeComment                 // A single or multiline comment
+	NodeTypeUseFlag                 // A use flag
```

Comment:
Are we worried about shifting the order of subsequent node types?

If not, I'll probably change some of how I'm doing my work.

Reply:
No, the order shouldn't matter here. All references are by names, and the tests all use the names too